### DISCUSSION TITLE
Epoptes and SquashFS image (DebianEdu Skolelinux 11.5)

### DISCUSSION THREAD

**Discussion by SuperSpin322** [epoptes/epoptes#173] [General]
Hello, we're using SquashFS image for clients (DebianEdu Skolelinux 11.5)

- It is somehow compatible that epoptes functions with squashfs?

- How should I do so that the client key is contained in the created image?

- Can you copy it from one folder to another on the computer from which the image is created (which is also an epoptes server)?

Than You!!

**Comment by alkisg** [comment]
Sure, Epoptes was designed for LTSP (=squashfs image). It works out of the box there.
AFAIK DebianEdu SkoleLinux uses LTSP underneath; if it doesn't work out of the box something isn't configured properly.

Unfortunately I've never used SkoleLinux so I'm not sure how it's organized, if it's using chroots, if it's chrootless etc.
The main file that needs to be in the image is /etc/epoptes/server.crt.
If you're having difficulties, it would be best to come to the chat room for live support via screen sharing, so that I see what's going on in your setup: https://ltsp.org/guides/chat-room/
